test/ttmlir/Dialect/TTIR/select/select_tests_negative.mlir

```diff
@@ -0,0 +1,103 @@
+// RUN: not ttmlir-opt --split-input-file %s 2>&1 | FileCheck %s
```

Comment:
What does the `not` do in `RUN`?

Reply:
It's used for negative tests where you anticipate errors, so `not` will negate the error code of the mlir-opt tool to get success.